MarkdownFile.js

```diff
@@ -591,6 +591,16 @@ MarkdownFile.prototype._walk = function(node) {
         case 'emphasis':
         case 'strong':
             node.title && this._addTransUnit(node.title);
+            if (this.localizeLinks && node.url) {
+                var value = node.url;
+                var parts = trim(this.API, value);
+                // only localizable if there already is some localizable text
+                // or if this text contains anything that is not whitespace
+                if (parts.text) {
+                    this._addTransUnit(node.url);
```

Comment:
Let's add `node.title` as a context comment here. I think it might be useful in a sense of making it simpler to connect the url and its text.

Reply:
The text in the square brackets appears as children nodes in the parse tree, not as a title tag. The title tag is for the `emphasis` or `strong` node types. If it were easy to get the contents in the square brackets, then I would agree, but it would take a much bigger change to get that info.